.circleci/config.yml

```diff
@@ -391,9 +426,13 @@ commands:
 
     # Configurable build/test commands
 
-    access-token-xcconfig:
+    access-token-configs:
         steps:
-            - run: echo "MAPBOX_ACCESS_TOKEN = ${MAPBOX_ACCESS_TOKEN}" > ./developer.xcconfig
+            - run:
+                name: Save mapbox access token to configs
+                command: |
+                    echo "MAPBOX_ACCESS_TOKEN = ${MAPBOX_ACCESS_TOKEN}" > ./developer.xcconfig
+                    echo "${MAPBOX_ACCESS_TOKEN}" > ~/.mapbox
```

Comment:
Ideally, we would choose one way or the other. We can address in a follow-up.

Reply:
We can fix it with a further PR.